### DISCUSSION TITLE
Typos etc

### DISCUSSION THREAD

**Discussion by lrljoe** [coderflexx/laravel-ticket#37] [General]
Going through the code. There are a few typos for config references etc.

E.g.
```
    'user_foreing_id' => 'user_id',
```
Should really be
```
    'user_foreign_id' => 'user_id',
```
With relevant code updates etc.

Not sure how widespread this package is at the moment, as fixing the typos may be a breaking change.

Likewise, I'd probably like to do a PR to change to morphs rather than referencing user_id, and allow uuid references for those too potentially (as an option rather than default)

Thoughts?

**Comment by ousid** [comment]
Hi, @lrljoe 

Thank you for taking the time, and notice the typos.

Yes, you can make a PR, and search for the used `user_id`, called by the `config` helper, then replace it. Besides, you can always run the tests to make sure everything is good to go.

The initial __DB Design__ for this package, was not to do any `polimorphic` relationship, just simple relations, and if we are changing it now, I'm afraid, that's a breaking change discussion.

PR's, always welcome!
